src/main/resources/migrations/00035_alter_table_events_add_external_id.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE events ADD COLUMN external_id VARCHAR(26);
```

Comment:
Due to the way our database transactions work (we have to explicitly initiate a migration: they don’t usually run automatically on start-up), we tend to prefer to have a separate PR with the database changes. Otherwise, the new code will be running alongside the old database schema and won’t work until the migration is done. Separating the migration into a prior PR avoids this problem.

Reply:
https://github.com/alphagov/pay-direct-debit-connector/pull/218